Source/WebCore/rendering/updating/RenderTreeUpdaterViewTransition.cpp

```diff
@@ -81,7 +81,7 @@ void RenderTreeUpdater::ViewTransition::updatePseudoElementTree(RenderElement& d
         newViewTransitionRoot->initializeStyle();
         documentElementRenderer.view().setViewTransitionRoot(*newViewTransitionRoot.get());
         viewTransitionRoot = newViewTransitionRoot.get();
-        m_updater.m_builder.attach(documentElementRenderer, WTFMove(newViewTransitionRoot));
+        m_updater.m_builder.attach(*documentElementRenderer.parent(), WTFMove(newViewTransitionRoot));
```

Comment:
I'm not sure we need this change, we'll need special code to change z-order anyway.

Reply:
If the root element has a view transition, then we reparent that layer to be a child of the `::view-transition-new` pseudo. That pseudo can't be a descendant of the root element, otherwise it's cyclical, so we need the pseudo tree to be attached the view.